build/syncset.go

```diff
@@ -339,12 +315,7 @@ func createValidatingWebhookConfiguration(hook webhooks.Webhook) admissionregv1.
 			Name: fmt.Sprintf("sre-%s", hook.Name()),
 
 			Annotations: map[string]string{
-				"managed.openshift.io/inject-cabundle-from": fmt.Sprintf("%s/webhook-cert", *namespace),
-				// TODO(lseelye): Leave out until Until 4.4 when openshift-ca-operator
-				// supports ValidatingWebhookConfigurations, so as to not fight against
-				// that operator once 4.4 lands.
-				//	// For openshift/service-ca-operator
-				//	"service.beta.openshift.io/inject-cabundle": "true",
+				"service.beta.openshift.io/inject-cabundle": "true",
```

Comment:
I would add a comment here explaining what the `service.beta.openshift.io/inject-cabundle` annotation will trigger.

Reply:
Can do. I fell into the trap of "well the annotation is obvious" but while it does say "inject cert" it is not clear on the face of it what that implies, and what is doing injection. I will update and provide a comment.